nautobot/docs/user-guides/git-data-source.md

```diff
@@ -0,0 +1,269 @@
+# Git as a Data Source
+
+The "Git™ as a Data Source" feature was developed to provide the ability to populate existing data, templates, scripts, and much more into Nautobot; while leveraging the benefits that tools such as GitHub and GitLab already provide, including issue tracking, discussions, pipelines, and approvals.
```

Comment:
Any reason you're including TM?

Reply:
Might be over thinking this one, but read https://git-scm.com/about/trademark section 2.1.  I maybe confusing whether its necessary or not.